scripts/style_mechanics_normalize.py

```
@@ -0,0 +1,22 @@
+import re
+
+JP_TO_EN = str.maketrans({"、": ", ", "。": ". ", "「": '"', "」": '"', "／": "/", "　": " "})
+
+def normalize_punct(s: str) -> str:
+    s = (s or "").translate(JP_TO_EN)
+    s = s.replace(" ,", ",").replace(" .", ".").replace(" %", "%")
+    s = re.sub(r"(\d)\s*-\s*(\d)", r"\1–\2", s)  # 5-10 → 5–10
```

Ruff (0.12.2)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)